types/model.go

```diff
@@ -2,6 +2,14 @@ package types
 
 import "time"
 
+// Secret for underlying orchestrator
+type Secret struct {
+	Name      string `json:"name"`
+	Namespace string `json:"namespace,omitempty"`
+	Value     string `json:"value,omitempty"`
+	RawValue  []byte `json:"rawValue,omitempty"`
```

Comment:
It would be very helpful for this to field in particaular to have a doc string, because it must be base64 encoded when submitted